pkg/certificate/manager_test.go

```diff
@@ -88,8 +88,6 @@ var _ = Describe("Test Rotor", func() {
 			close(done)
 
 			fmt.Printf("It took %+v to rotate certificate %s\n", time.Since(start), cn)
-			//stop rotation
-			certManager.stop()
```

Comment:
If cert rotation still needs to stop here, that would look like
```go
close(certManager.certRotation)
```

Or if the existing `stop` channel is used to initialize this, then that will already be closed so you don't need to explicitly stop it here.